woocommerce/woocommerce-ios
# Download Arabic and Hebrew metadata

Adds Arabic and Hebrew localization downloads when finalizing the release – this will ensure these localizations are present to be uploaded to App Store Connect.

**To Test**
- Run `./fastlane/download_metadata.swift` from the project's root directory
- See that these files haven't changed (others likely will – it's ok to ignore that)

**Update release notes:**
